src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs

```diff
@@ -185,7 +204,7 @@ private void ProcessRequests(ITestRequestManager testRequestManager)
                                 var testRunPayload =
                                     this.communicationManager.DeserializePayload<TestRunRequestPayload>(
                                         message);
-                                this.StartTestRun(testRunPayload, testRequestManager, skipTestHostLaunch: true);
+                                this.StartTestRun(testRunPayload, testRequestManager, shouldLaunchTesthost: true);
```

Comment:
is this correct? it seems that the parameter changed the meaning, but we didn't change the value

Reply:
Good point, thanks. The meaning was wrong before but now it accurately reflects the intent. When the parameter is set on false we use no custom launcher, but we do when it's set on true.